go/ct/driver/run.go

```diff
@@ -44,33 +44,16 @@ var RunCmd = cli.Command{
 	Usage:     "Run Conformance Tests on an EVM implementation",
 	ArgsUsage: "<EVM>",
 	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "filter",
-			Usage: "run only rules which name matches the given regex",
-			Value: ".*",
-		},
-		&cli.IntFlag{
-			Name:  "jobs",
-			Usage: "number of jobs run simultaneously",
-			Value: runtime.NumCPU(),
-		},
+		cliUtils.FilterFlag,
+		cliUtils.JobsFlag,
+		cliUtils.SeedFlag,
+		cliUtils.CpuProfileFlag,
+		cliUtils.FullModeFlag, // < TODO: make every run a full mode once tests pass
 		&cli.IntFlag{
 			Name:  "max-errors",
 			Usage: "aborts testing after the given number of issues",
 			Value: 100,
 		},
```

Comment:
Why didn't you add the `max-errors` flag to the cli/utils?

Reply:
Because it is only used in this file. 
We will come back to the driver in the future to rename commands and add more filters. I will leave it for now.